etc/schema.json

```diff
@@ -3465,6 +3465,10 @@
             "properties": {
                 "linktype": {
                     "type": "integer"
+                },
```

Comment:
Any thoughts on deprecating this field given its non-portable?

Reply:
How is it not portable ? (the link type in a pcap is encoded as some integer, is it not ?)

Reply:
The mapping between the linktype value and name is implementation dependent. Most of the mappings are the same but not all.

Reply:
So, when reading a pcap, we read its linktype as an integer, but if we do not know on which implementation it was captured, this linktype may mean different things, correct ?

So, still good to log the linktype as it is the thing we know from the pcap file... correct ?

Reply:
Yes; we use the pcap library on the host running Suri to interpret the linktype value

Reply:
It seems that we don't document `packet_info` our docs (or I couldn't find it). So I'm thinking we can add a ticket to document it, and include in the task registering this aspect of the linktype...